Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
I don't know what this is actually used for, but it currently returns either the tile content bounding volume or the tile bounding volume. Should it be called `contentsBoundingVolume` even if it may return the tile's bounding volume?